content/articles/recover-deleted-files-with-foremost/index.md

```diff
@@ -0,0 +1,121 @@
+### **Recovering deleted files using foremost**
+
+Unintentional deletion of important files can be tragic. This has led to operating systems having a restore point. In Linux, the restore point is the trash. In Windows, the restore point is the recycle bin. Sometimes we delete these files in the restore points too. At this point, we opt to visit data recovery centers or buy expensive data recovery software.
+
+### Introduction
+
+[**Foremost**](https://pkgs.org/download/foremost) is a free data retrieval software. You can download it on the internet. It operates on Linux operating systems. Throughout this tutorial, we will be using Kali Linux. Anyone using any other Linux distribution can follow through.
+
+In this tutorial, we shall be looking at the use of **foremost** in data recovery of lost/deleted files from a USB drive. We shall then go through how to recover deleted files from a whole hard disk. Finally, we will learn how to recover deleted files from a particular partition of a hard drive.
+
+### Prerequisites
+
+To understand this article a reader needs to have:
+
+- Kali Linux or any other Linux distribution installed.
+- Prior knowledge of the basic commands in the Linux terminal.
+
+Check the following article to understand the prerequisites.
+[Getting started with kali linux](https://www.section.io/engineering-education/getting-started-with-kali-linux/)
+
+### Installing foremost in Kali Linux
+
+Foremost comes pre-installed in most Linux distributions. If it is not installed, type the following command in the terminal. After executing the command **foremost** will be installed.
+
+```bash
+sudo apt install foremost
+```
+
+### Foremost package tools
+
+**Foremost** package includes some flags. We use the following command to display them.
+
+```bash
+foremost -h
+```
+
+The command displays all the flags included and their use. Below is a screenshot of its display.
+
+![Tools](/engineering-education/recover-deleted-files-with-foremost/tools.png)
+
+### 1. Recovering files from a USB drive
+
+Attach the USB to the computer. List the drives on your computer to determine the path of your USB drive with the following command.
+
+```bash
+sudo fdisk -l
+```
+
+The above command gives the following output.
+
+![List Drives](/engineering-education/recover-deleted-files-with-foremost/drives.png)
+
+Scroll down to where your USB is located and copy the file path. To start the file recovery process, use the command below:
+
+```bash
+sudo foremost -v -t jpg,pdf,mp4,exe -q -i /dev/sdc -o /home/Desktop/recover -T
+```
```

Comment:
Is this file path bound to change depending on the users machine? Let it be clear to the reader